packages/react/scroll-area/src/ScrollArea.tsx

```diff
@@ -715,6 +715,7 @@ const ScrollAreaScrollbarImpl = React.forwardRef<
             // so we remove text selection manually when scrolling
             prevWebkitUserSelectRef.current = document.body.style.webkitUserSelect;
             document.body.style.webkitUserSelect = 'none';
+            if (context.viewport) context.viewport.style.scrollBehavior = 'auto';
```

Comment:
Do we also need to do something to address the wheel interaction mentioned in the issue?

Reply:
No, I tested using my mouse wheel, and using my trackpad and there was no issue with those.
Similarly, I tested with touch drag on my iPhone and it was fine too.

This makes sense to me as it makes sense that `scroll-behavior: smooth` takes that into account already.